src/App.tsx

```diff
@@ -1,24 +1,35 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+/* eslint-disable react-hooks/rules-of-hooks */
+import React, { useContext } from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import { TodoFooter } from './components/footer/TodoFooter';
+import { Errors } from './components/Errors/Errors';
+import { TodoRenderList } from './components/main/TodoRenderList';
+import { TodoHeader } from './components/header/Header';
+import { appContext } from './components/Context/Context';
 
 export const App: React.FC = () => {
+  const { USER_ID, todos } = useContext(appContext);
```

Comment:
You can create a custom hook so you don't need to pass context every time to `useContext`
```typescript
const useAppContext = () => useContext(appContext);
```